index.md

```diff
@@ -301,6 +302,7 @@ color by default via `NO_COLOR`.
 | [Ruby (tests)](https://www.ruby-lang.org/) | `export RUBY_TESTOPTS=--color=never` |
 | [The Silver Searcher](https://geoff.greer.fm/ag/) | `ag --nocolor` ([Rejected `NO_COLOR` Request](https://github.com/ggreer/the_silver_searcher/pull/1207)) |
 | [Thor](http://whatisthor.com/) | `export THOR_SHELL=Basic` ([Docs](http://www.rubydoc.info/github/wycats/thor/Thor%2FBase.shell)) |
+| [tsfm](https://npmjs.com/package/tsfm) | `tsfm --no-color` |
```

Comment:
If the package supports `NO_COLOR`, it should not be listed in the bottom section of things that don't support it.

Reply:
Finished